### DISCUSSION TITLE
Nach Migration von Domain "hahm" auf "homematic_local" sind keine HM-RF-Geräte mehr in HA

### DISCUSSION THREAD

**Discussion by snookerap** [SukramJ/aiohomematic#1056] [Q&A] [answered]
Hi,
nachdem ich nun die Migration auf die neuere Domain vollzogen habe, werden keine Homematic-Classic-Geräte (HM-RF) mehr in HA bereitgestellt.
Den Port 2001 habe ich in der CCU freigegeben und in der Integration auch ausgewählt.
Mit der alten Domain hatte es wunderbar funktioniert.

Jemand noch eine Idee wo ich etwas einstellen kann?

Viele Grüße und danke vorab...

**Comment by SukramJ** [comment]
Von neuer Domain kann man wohl kaum noch sprechen.

Vielleicht solltest Du mal in das HA Log schauen (vollständiges Log) da könnten relevante Informationen zur Problemlösung drinstehen.

**Reply by snookerap** [reply]
Deswegen schrieb ich ja auch, die neuerer Domain... Bisher sah ich keinen Grund zu wechseln.. leider..

In den Logs kann ich nichts verdächtoges finden.
Nach was müsste ich Ausschau haletn?

**Reply by SukramJ** [reply, marked as answer]
Wenn in den logs nicht mit `homematic` auftaucht, dann mal den Cache mit dem Service (Entwicklerwerkzeuge) `Homematic(IP) Local: Clear the cache` löschen und HA neustarten

**Reply by snookerap** [reply]
Das war die Lösung, jetzt sind die Classic-Homematic-Geräte wieder da.

Danke... :-)
